File: TorrentAdder/SYNavigationController.m
//
//  SYNavigationController.m
//  TorrentAdder
//
//  Created by Stan Chevallier on 05/10/2015.
//  Copyright © 2015 Syan. All rights reserved.
//

#import "SYNavigationController.h"
#import "UIColor+SY.h"

@interface SYNavigationController ()

@end

@implementation SYNavigationController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self.navigationBar setBarTintColor:[UIColor lightBlueColor]];
    [self.navigationBar setTintColor:[UIColor whiteColor]];
    [self.navigationBar setBarStyle:UIBarStyleBlackOpaque];
    [self.navigationBar setTitleTextAttributes:@{NSForegroundColorAttributeName:[UIColor whiteColor]}];
}

@end

File: TorrentAdder/SYNavigationController.swift
//
//  SYNavigationController.swift
//  TorrentAdder
//
//  Created by Stanislas Chevallier on 28/11/2018.
//  Copyright © 2018 Syan. All rights reserved.
//

import UIKit

class SYNavigationController: UINavigationController {

    override func viewDidLoad() {
        super.viewDidLoad()

        navigationBar.setBackgroundImage(UIImage.sy_image(with: .lightBlue), for: .default)
        navigationBar.setBackgroundImage(UIImage.sy_image(with: .lightBlue), for: .compact)
        navigationBar.tintColor = .white
        navigationBar.barStyle = .blackOpaque
        navigationBar.titleTextAttributes = [.foregroundColor: UIColor.white]
    }
}
